### PR TITLE
Add Mullvad VPN module with extended functionality

### PR DESCRIPTION
**Pull request by totoroot**
There already is a script that simply displays the connection status to Mullvad VPN.

This is a script that toggles Mullvad VPN connection with `mullvad` command line client, shows the connection status and optionally the location of the VPN server currently connected to. It is possible to configure the module to cycle through available server locations on scroll too.

### CODE REVIEW

File: polybar-scripts/vpn-mullvad/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Script: vpn-mullvad
```
Comment (x70b1) [inline polybar-scripts/vpn-mullvad/README.md:1]: We should replace the existing script with your work.
Rename the script and directory to `vpn-mullvad-status`.

File: polybar-scripts/vpn-mullvad/vpn-mullvad.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+previous_country() {
+    list_country_shortcodes | grep "$(current_country)" -B1 | head -n 1 | awk '{printf $0}' 
+}
+
+next_country() {
+    list_country_shortcodes | grep "$(current_country)" -A1 | tail -n 1 | awk '{printf $0}' 
+}
+
+case $1 in
+	"status") connection_status;;
```
Comment (x70b1) [inline polybar-scripts/vpn-mullvad/vpn-mullvad.sh:32]: Flags starting with -- would be better.

File: polybar-scripts/vpn-mullvad/vpn-mullvad.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+current_country() {
+    mullvad status --location | grep Relay | cut -c8-9
+}
+
+list_country_shortcodes() {
+    mullvad relay list | grep -v '^[[:blank:]]' | cut -d '(' -f2 | cut -d ')' -f1 | sed '/^$/d'
+}
+
+# list_city_shortcodes() {
```
Comment (x70b1) [inline polybar-scripts/vpn-mullvad/vpn-mullvad.sh:15]: Do we still need this function?

File: polybar-scripts/vpn-mullvad/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+A script that toggles Mullvad VPN connection with `mullvad` command line client, shows the connection status and optionally the location of the VPN server currently connected to. It is possible to configure the module to cycle through available server locations on scroll too.
+
+![vpn-mullvad](screenshots/1.png)
```
Comment (x70b1) [inline polybar-scripts/vpn-mullvad/README.md:5]: Add the screenshots to the main README.md.

**Review by x70b1 — CHANGES_REQUESTED** [review]
Nice work! Lets replace vpn-mullvad-status.

**Comment by totoroot** [comment]
Closing this in favour of PR #383

**Pull request closed without merging**